graphitepager/worker.py

```diff
@@ -120,11 +120,11 @@ def run(args):
                     http_connect_timeout_s_ = http_connect_timeout_s,
                     http_read_timeout_s_    = http_read_timeout_s,
                 )
-            except requests.exceptions.RequestException as e:
+            except (ValueError, requests.exceptions.RequestException) as e:
```

Comment:
Any ValueError propagates to the top-level loop here, we'll report and continue with the rest of our work instead of crashing.